src/v/cloud_roles/tests/test_definitions.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2022 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+#pragma once
```

Comment:
Empty file. No new line at the end.

Reply:
I think I messed up this file during restructuring commits, because the final version of the file does have content, I will remove it from the old commits where it was not required.

Reply:
removed the file from old commit, now it is introduced in the tests commit